ideascube/conf/base.py

```diff
@@ -105,27 +105,15 @@
 
 USE_TZ = True
 
-# Languages that will be available in all forms.
+# Languages that will be available as UI translations
 AVAILABLE_LANGUAGES = (
```

Comment:
I thought about that, yes, but thought it might be too much, maybe you had a reason to have it split between `base` and `settings`... Then I forgot to ask. :smile: 

But if there isn't such a reason, then yes, let's move it all to `base`. Will do when I get a chance.